ocfweb/account/register.py

```diff
@@ -42,14 +41,15 @@ def request_account(request: HttpRequest) -> Union[HttpResponseRedirect, HttpRes
     status = 'new_request'
 
     existing_accounts = search.users_by_calnet_uid(calnet_uid)
-    eligible_group_signatories = groups_by_student_signat(calnet_uid)
+    eligible_signatory_groups = groups_by_student_signat(calnet_uid)
 
+    # disallow creation forr groups already have accounts
     existing_group_accounts = {}
-    for group_oid in list(eligible_group_signatories.keys()):
-        if group_by_oid(group_oid) and group_oid not in map(operator.itemgetter(0), TEST_GROUP_ACCOUNTS):
-            existing_group_accounts[group_oid] = eligible_group_signatories.pop(group_oid)
+    for group_oid in list(eligible_signatory_groups.keys()):
+        if len(group_by_oid(group_oid)['accounts']) and group_oid not in [group[0] for group in TEST_GROUP_ACCOUNTS]:
```

Comment:
empty list is already falsy (no need for len). thanks for the clarification on this tho